Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -259,13 +277,56 @@ public class Klaviyo: NSObject  {
                 , "Property values must be of NSString, NSNumber, NSNull, NSDictionary, NSDate, or NSURL. Got: \(String(describing: properties[k as! NSCopying]))")
         }
     }
+}
+
+
+struct KlaviyoSDK {
```

Comment:
I'm mixing up languages here -- does this need to be marked internal or private so they can't instantiate it themselves or is that what is is by default in swift.

Reply:
Internal is default in swift.